app/themes/ukis/templates/introduction.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'introduction.html' %}
+
+{% block header %}
+  {% include 'partials/header.html' %}
+  {% with is_hero = true %}
+    {% include theme('partials/topbar.html') %}
+  {% endwith %}
+{% endblock %}
+
+{% block business_details %}
+  <div class="u-mb-xs">
+    {% if meta.survey.region_code == 'GB-NI' %}
+      <img src="{{ theme_static('logo-dof.png') }}" alt="Department of Finance" width="183" />
+    {% else %}
+      <img src="{{ theme_static('logo-beis.png') }}" alt="Department for Business, Energy and Industrial Strategy" width="660" />
+    {% endif %}
+  </div>
+
+  {{super()}}
```

Comment:
By including the default `business_details` block, the text 

> Your response is legally required

 is output, the design on the Trello card uses the text

> Your response to this survey is of great value to government, whether or not you carry out innovation activities